developers.properties

```diff
@@ -0,0 +1,6 @@
+# This file is here for 'hapi', because that "product" is missing the toplevel folder
+# (it contains the 'hapi' module directly)
+hedera-base@swirldslabs.com=Hedera Base Team
+hedera-services@swirldslabs.com=Hedera Services Team
+hedera-smart-contracts@swirldslabs.com=Hedera Smart Contracts Team
+release-engineering@swirldslabs.com=Release Engineering Team
```

Comment:
For symmetry, I would suggest to move `hapi` one level up into `hapi/hapi`. Even if it is just one "Module" in the "Product". (In a separate PR.)